Where_Is_My_Wife/Assets/_Main/Scripts/Input/InputEventManager.cs

```diff
@@ -0,0 +1,246 @@
+using System;
+using UniRx;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using WIMW.Input;
+using Zenject;
+
+namespace WhereIsMyWife.Managers
+{
+    public partial class InputEventManager : IPlayerInputEvent
+    {
+        private readonly Subject<Unit> _jumpStartSubject = new Subject<Unit>();
+        private readonly Subject<Unit> _jumpEndSubject = new Subject<Unit>();
+        private readonly Subject<float> _runSubject = new Subject<float>();
+        private readonly Subject<Vector2> _dashSubject = new Subject<Vector2>();
+        private readonly Subject<Vector2> _useItemSubject = new Subject<Vector2>();
+        private readonly Subject<Unit> _hookStartSubject = new Subject<Unit>();
+        private readonly Subject<Unit> _hookEndSubject = new Subject<Unit>();
+        private readonly Subject<Unit> _lookUpSubject = new Subject<Unit>();
+        private readonly Subject<bool> _lookDownSubject = new Subject<bool>();
```

Comment:
Change this to `Action` or `UnityAction`

Reply:
ad2bfe96fff939d10ba43a2d9cb8eedc1254f38b

Reply:
bff16c4b5b11853d49da3ae33e8035b462b7ec6e